app/invocation/invocation_execution_card.tsx

```diff
@@ -275,12 +274,8 @@ export default class ExecutionCardComponent extends React.Component {
 
     return (
       <div>
-        <InvocationFilterComponent
-          hash={"#execution"}
-          search={this.props.search}
-          placeholder={"Filter by digest or command..."}
-        />
         <div className={`card expanded`}>
+          <img className="icon" src="/image/cloud-regular.svg" />
```

Comment:
I'm not seeing this icon in the screenshots - where does this show up?

Reply:
Realized that the screenshots are from before i added the icon but this is what it looks like after 
<img width="1174" alt="Screen Shot 2021-08-19 at 4 20 36 PM" src="https://user-images.githubusercontent.com/17104765/130138465-c15bf4b2-6214-4df2-a7f0-f958e6cbad2f.png">

Reply:
Could we maybe just leave it out? It pushes all of the content to the right.